modules/network-observability-loki-secret.adoc

```diff
@@ -6,7 +6,7 @@
 [id="network-observability-loki-secret_{context}"]
 = Creating a secret for Loki storage
 
-The {loki-op} supports a few log storage options, such as AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation. The following example shows how to create a secret for AWS S3 storage. The secret created in this example, `loki-s3`, is referenced in "Creating a LokiStack resource". You can create this secret in the web console or CLI.
+The {loki-op} supports a few log storage options, such as AWS S3, Google Cloud Storage, Azure, Swift, Minio, {rh-storage}. The following example shows how to create a secret for AWS S3 storage. The secret created in this example, `loki-s3`, is referenced in "Creating a LokiStack resource". You can create this secret in the web console or CLI.
 
 . Using the web console, navigate to the *Project* -> *All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.
```

Comment:
Might be out of scope, but consider stepping out "Name the project `netobserv` and click *Create*." in its own step as step 1 has a few steps.